packages/components/package.json

```diff
@@ -83,6 +83,7 @@
     "@typescript-eslint/eslint-plugin": "^8.5.0",
     "@typescript-eslint/parser": "^8.5.0",
     "babel-plugin-ember-template-compilation": "^2.2.4",
+    "clipboard-polyfill": "^4.1.0",
```

Comment:
issue: this should be in `dependencies` if consumers are to rely on it, yes?

Reply:
oh! yes thats right - will update.